storage/remote/read_test.go

```diff
@@ -22,6 +23,45 @@ import (
 	"github.com/prometheus/prometheus/storage/metric"
 )
 
+func TestValidateNameAndLabels(t *testing.T) {
+
```

Comment:
Nitpick: It's not common in stdlib or Prometheus golang code to include empty lines at the beginning or end of functions.